mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp

```diff
@@ -289,6 +289,7 @@ USER_DEFINED_WRAPPER(int, Waitany, (int) count,
         *local_index = i;
         return retval;
       }
+      DMTCP_PLUGIN_ENABLE_CKPT();
```

Comment:
There are multiple returns. I'm afraid that the function can return before enabling CKPT.

Reply:
Yes, I have the same concern here.

Reply:
We can use break instead of return.

Reply:
Good catch. Fixed.